packages/babel-preset-env/test/fixtures/debug/entry-no-corejs-shippedProposals/input.mjs

```diff
@@ -1 +1 @@
-import '@babel/polyfill';
+import 'core-js';
```

Comment:
The CoreJS3EntryPlugin has thrown warning for `@babel/polyfill` imports and it does _not_ replace these imports to `core-js` submodules. I think this behaviour is acceptable and we can preserve it in Babel 8.

The test case is about when `core-js` is _not_ specified and `@babel/polyfill` is imported, it should be replaced via the usage plugin by a series of `core-js` submodule imports. Since we have also changed the `core-js` defaults to 3, we shall also revise this case to showcase that `import "core-js"` will be replaced, too.